pallets/loans/src/benchmarking.rs

```diff
@@ -310,6 +302,37 @@ fn pool_account<T: pallet_pool_system::Config>(pool_id: T::PoolId) -> T::Account
 	PoolLocator { pool_id }.into_account_truncating()
 }
 
+// Populate interest rates for worst-case lookup of our actual rate
+fn populate_bench_storage<T: Config>(
+	pool_id: PoolIdOf<T>,
+	loans: u32,
+	rates: u32,
+) -> Option<AssetOf<T>>
+where
+	<T as LoanConfig>::Rate: From<Rate>,
+	<T as LoanConfig>::LoanId: From<u32>,
+	<T as InterestAccrualConfig>::InterestRate: From<Rate>,
+	<T as pallet_balances::Config>::Balance: From<u128>,
+	<T as pallet_uniques::Config>::CollectionId: From<u64>,
+	<T as pallet_pool_system::Config>::PoolId: Into<u64> + IsType<PoolIdOf<T>>,
+{
+	let mut collateral = None;
+	for idx in 1..rates {
+		let rate = Rate::saturating_from_rational(idx, 5000).into();
```

Comment:
Since `5000` seems to be the default rate quotient for the benchmarks as it is used all three times, I think we should declare this a constant.